lib/basedriver/commands/find.js

```diff
@@ -32,6 +41,15 @@ helpers.findElOrElsWithProcessing = async function (strategy, selector, mult, co
 };
 
 commands.findElement = async function (strategy, selector) {
+  if (strategy === IMAGE_STRATEGY) {
+    // in the singular findElement only, we can find by image, which is
+    // a special case. In any other find command (including
+    // findElement(s)FromElement), this doesn't make sense, so use of the
+    // -image locator strategy won't pass the strategy validation filter as is
+    // desired
+    return await this.findByImage(selector);
```

Comment:
I would say there's still a difference, because in case of multiple elements we need to return an empty array or throw an exception if only one element is expected

Reply:
But in case of multiple elements (or anything other than strict `findElement`) it will throw an error saying there is no such strategy (https://github.com/appium/appium-base-driver/blob/master/lib/basedriver/commands/find.js#L19).